Userland/Utilities/chown.cpp

```diff
@@ -66,12 +71,30 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         }
     }
 
-    for (auto const& path : paths) {
-        if (dont_follow_symlinks) {
+    Function<ErrorOr<void>(StringView const&)> update_path_owner = [&](StringView const& path) -> ErrorOr<void> {
```

Comment:
No StringView by const reference pls

Reply:
see https://github.com/SerenityOS/serenity/commit/8b1108e4858f797c9216dc8ae4a3918ad50c73b4

Reply:
Looks like one slipped through then just recently, but at least this one is fixed now. :^)